# 
# @brief    : custom mavlink msg, message id="152" name="FIXED_TARGET_POSITION_P2M"
# @time     : 2018/09/13
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
#

int32 measure_enable